.travis.yml

```diff
@@ -3,8 +3,6 @@ node_js:
   - '10'
   - '12'
 
-dist: trusty
```

Comment:
This dist is EOL and is no longer supported by Chrome. That was the reason for the failing builds.